edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -153,22 +167,22 @@ const LectureScreen: ApplicationRoute = () => {
                 isLandscape && ScreenOrientation.unlockAsync();
                 setIsFullscreen(!isFullscreen);
             }}>
-                <Icon size={'xl'} name={isFullscreen ? 'contract-outline' : 'expand-outline'} dark='text' testID='fullscreen-toggle'></Icon>
+                <Icon size={'xl'} name={isFullscreen ? 'contract-outline' : 'expand-outline'} color='text' testID='fullscreen-toggle'></Icon>
             </TTouchableOpacity>
         </TView >
     );
 
     const ContentView = (widthPercent: string, heightPercent: string) => (
         <TView flexDirection='column' mr={'xl'} style={{ width: widthPercent as DimensionValue, height: heightPercent as DimensionValue }}>
-            <TScrollView b={'sm'} mt={25} mr={'md'} ml={'md'} radius={'lg'} flex={1}>
+            {/*<TScrollView b={'sm'} mt={25} mr={'md'} ml={'md'} radius={'lg'} flex={1}>
                 {currentLecture.audioTranscript?.[currentPage] ? (
                     <TText pl={'sm'} pr={'sm'}>{currentLecture.audioTranscript[currentPage]}</TText>
                 ) : (
                     <TText pt={'sm'} pl={'sm'} pr={'sm'} color='overlay0'>
                         {t(`showtime:lecturer_transcript_deftxt`)}
                     </TText>
                 )}
-            </TScrollView>
+            </TScrollView>*/}
```

Comment:
Why have you commented out the place where we display the live lecturer transcript ?
Do you display it in another place ? 
Just watch out as this code is behind from the main version.

Reply:
Yes since you have recently modified the way audioTranscript was formated it was having some bugs in my version because the firebase was updated, however I haven't merged with your current code yet so I kept it commented for while I was testing and would merge once I finished this PR by resolving conflicts now

Reply:
That makes totally sens. You did the right thing, sorry for the incoveniance. I've checkout the merge resolve. It seems fitting our both changes perfectly.